### PR TITLE
Clean up of the folder structure

### PR DESCRIPTION
**Pull request by tom-reinders**

### CODE REVIEW

File: nginx/Dockerfile [hunk 1]
```diff
@@ -13,8 +13,6 @@ ADD nginx/docker/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 ADD nginx/docker/php-fpm-app.conf /usr/local/etc/php-fpm.d/php-fpm-app.conf
 ADD symfony.ini /usr/local/etc/php/conf.d/symfony.ini
 
-RUN sed -i 's/user www-data;//g' /etc/nginx/nginx.conf
```
Comment (wslaghekke) [inline nginx/Dockerfile:16]: ?

File: src/index.ts [hunk 2]
```diff
@@ -136,9 +136,9 @@ function getTags(client: {
 }
 
 async function getNodeLtsVersions() {
-    const response = await axios.get<NodeVersion[]>('https://nodejs.org/dist/index.json');
+    const response = await axios.get<{ version: string, lts: string|boolean }[]>('https://nodejs.org/dist/index.json');
     const groupedVersion = groupBy(response.data.filter(it => it.lts), 'lts');
-    const versions: { version: string, major: string, lts: string }[] = [];
+    const versions: NodeVersion[] = [];
```
Comment (wslaghekke) [inline src/index.ts:141]: ?

File: nginx/Dockerfile (hunk 1 above)
Comment (tom-reinders) [reply nginx/Dockerfile:16]: This line is always remove by this sed, and is never used, so i just removed the line out of the original file

File: src/index.ts (hunk 2 above)
Comment (tom-reinders) [reply src/index.ts:141]: NodeVersion is the same as { version: string, major: string, lts: string } where as on line 139 the this was wrong and https://nodejs.org/dist/index.json never resulted a major value.

**Review by wslaghekke — APPROVED** [review]

**Pull request merged**